README.md

```diff
@@ -11,53 +11,238 @@ Pour faire ce codelab, vous avez besoin des outils suivants :
+```
+npm run dev
+```
+
+### PW2 - Composants
+
+#### Intégration dans App.vue
+Nous sommes maintenant prêt pour intégrer notre site statique dans l'application qui vient d'être générée.
+
+Vous allez tout d'abord intégré le site statique dans l'application:
+    - les éléments à l'extérieur de `<body>` doivent se trouver dans le fichier index.html
+    - le contenu de `<body>` doit être inséré dans le composant App.vue
+    - (optionnel) vous pouvez déplacer le style du fichier `style.css` dans le composant `App.vue`
+
+#### Création des premiers composants
+
+Vous allez maintenant créer vos premiers composants Vue.
+
+- Créer un composant `search.vue` qui va contenir le formulaire de recherche. Mettre à jour App.vue pour définir ce composant de manière local.
+
+- Créer un composant `result.vue` qui va contenir le résultat de la recherche. Ce composant aura comme donnée une liste statique d'éléments. Ce composant affichera une card pour chaque élément en accédant directement au tableau via l'index.
+```javascript
+pokemons: [{
+    name: 'Bulbasaur',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png',
+}, {
+    name: 'Ivysaur',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/2.png',
+}, {
+    name: 'Venusaur',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/3.png',
+}, {
+    name: 'Charmander',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/4.png',
+}, {
+    name: 'Charmeleon',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/5.png',
+}, {
+    name: 'Charizard',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/6.png',
+}, {
+    name: 'Squirtle',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/8.png',
+}, {
+    name: 'Wartortle',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/9.png',
+}, {
+    name: 'Blastoise',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/10.png',
+}, {
+    name: 'Caterpie',
+    image: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png',
+}]
+```
+```html
+{{ pokemons[0].name }}
+```
+N'oubliez pas de mettre à jour App.vue pour importer ce nouveau composant.
+
+- On va maintenant extraire les cards de chaque pokemon dans un composant dédié `card.vue`. Ce composant prendre deux propriétées:
+    - `name` qui sera le nom du pokemon
+    - `image`qui sera l'URL de l'image à affichée
+Ré-écriver le composant `result.vue` pour utiliser ce nouveau composant cards
+
+- *(bonus)* inscrivez le composant `result.vue` de manière global.
+
+
+### PW3 - Directives
+
+
+Le composant `search.vue` est actuellement qu'un simple champ de formulaire. Nous allons le rendre un peu plus intéractif en lui ajoutant certains comportement:

```

Comment:
~comportement~ `comportement`

Reply:
i don't see the difference there ? You mean comportement*S* ?